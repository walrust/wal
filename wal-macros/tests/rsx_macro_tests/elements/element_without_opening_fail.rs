use wal_macros::rsx;

fn main() {
    rsx! { </div> };
    rsx! { </span> };
    rsx! { </> };
    rsx! { <div></span> };
    rsx! { <div><span></div></span></div> };
}
