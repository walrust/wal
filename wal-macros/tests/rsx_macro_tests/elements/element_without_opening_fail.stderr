error: This closing tag does not have a corresponding opening tag. (hint: try adding `<div>`)
 --> tests/rsx_macro_tests/elements/element_without_opening_fail.rs:4:13
  |
4 |     rsx! { </div> };
  |             ^^^^^^

error: This closing tag does not have a corresponding opening tag. (hint: try adding `<span>`)
 --> tests/rsx_macro_tests/elements/element_without_opening_fail.rs:5:13
  |
5 |     rsx! { </span> };
  |             ^^^^^^^

error: This closing fragment does not have a corresponding opening fragment. (hint: try adding `<>`)
 --> tests/rsx_macro_tests/elements/element_without_opening_fail.rs:6:13
  |
6 |     rsx! { </> };
  |             ^^^

error: This closing tag does not have a corresponding opening tag. (hint: try adding `<span>`)
 --> tests/rsx_macro_tests/elements/element_without_opening_fail.rs:7:18
  |
7 |     rsx! { <div></span> };
  |                  ^^^^^^^

error: This closing tag does not have a corresponding opening tag. (hint: try adding `<div>`)
 --> tests/rsx_macro_tests/elements/element_without_opening_fail.rs:8:24
  |
8 |     rsx! { <div><span></div></span></div> };
  |                        ^^^^^^
